mappings/net/minecraft/block/BedrockBlock.mapping

```diff
@@ -0,0 +1 @@
+CLASS blu net/minecraft/block/BedrockBlock
```

Comment:
Despite only being used for Bedrock, it's perfectly usable as a generic spawnproof block and has nothing specific to Bedrock other than the spawnproof part. It would look rather odd if someone made use of a class called `BedrockBlock` for some breakable spawnproof block in their mod.

Reply:
Based on the obfuscated name (in alphabetical order relative to other block classes) the Mojang name starts with a `B`, so it's probably `BedrockBlock`.

It would be a really bad idea to use that in a mod since mojang may add a method at any point.

It also doesn't really makes sense to have a "spawnproof block" class, since unlike the other block classes, "spawnproof" isn't a kind of block (glass, log, etc.).

Reply:
Hmm, good points. `BedrockBlock` does seem fine in that case.